git/gitgithubclient.py

```diff
@@ -40,7 +39,7 @@ def _commitData(self, commit):
 			"message": commit.commit.message
 		}
 
-	def commits(self, branch, since = 0, to = datetime.datetime.now() + datetime.timedelta(hours=24)):
+	def commits(self, branch, since=0, to=0):
```

Comment:
You are right, I've forgotten to add two lines:
else:
   to = datetime.datetime.fromtimestamp(to)
But for the previous comment, I tried to avoid having had to make timestamp from datetime and then changing it back (since Github.get_commits recieves datetime).